test-browser/tests/remixd.js

```diff
@@ -52,15 +52,20 @@ module.exports = {
       when a relative import is used (i.e import "openzeppelin-solidity/contracts/math/SafeMath.sol")
       remix (as well as truffle) try to resolve it against the node_modules and installed_contracts folder.
     */
-    browser
+    const browserName = browser.options.desiredCapabilities.browserName
+
+    return browserName === 'chrome' ? browser
       .waitForElementVisible('#icon-panel', 2000)
       .clickLaunchIcon('fileExplorers')
       .addFile('test_import_node_modules.sol', sources[3]['browser/test_import_node_modules.sol'])
       .clickLaunchIcon('solidity')
       .testContracts('test_import_node_modules.sol', sources[3]['browser/test_import_node_modules.sol'], ['SafeMath'])
+      : ''
   },
   'Import from node_modules and reference a github import': function (browser) {
-    browser
+    const browserName = browser.options.desiredCapabilities.browserName
+
+    return browserName === 'chrome' ? browser
```

Comment:
you don't run this test for firefox?

Reply:
Yes, the test is dependent on the previous tests you disabled for firefox. It also fails for chrome too when the beginning tests are disabled.